### ISSUE TITLE
I could not install the ClusterR

### ISSUE THREAD

**Issue by Da-Bean** [mlampros/ClusterR#9]
Hi, i`m Da-Bin Lee in Korea.

I tried install the ClusterR but i could not install with error message "this package depend on jpeg, png, OpenImage ,etc".
So i think that i have to install these package, and then i tried install the 'tiff', 'jpeg' but i could not install these with error message "this package not exist".

Please help me

**Comment by mlampros** [comment]
Hi @Da-Bean,

in which operating system (Linux, Macintosh, Windows) do you try to install the ClusterR package?
You receive these errors because the ClusterR package [imports](https://github.com/mlampros/ClusterR/blob/master/DESCRIPTION) the OpenImageR package.
I use R in a Linux-like operating system and In order to install the [OpenImageR](https://github.com/mlampros/OpenImageR) package I have first to do the following installs from the console,

```R

sudo apt-get install -y libfftw3-dev libtiff5-dev
sudo apt-get install libjpeg-dev
sudo apt-get install libpng-dev

```

then I continue with the installation of the package.

**Comment by Da-Bean** [comment]
Hi

I have a question one thing about the MiniBatchkmeans in ClusterR.
I used MiniBatchKmeans for data.
So, I obtained output, WCSS_per_cluster, centroids, etc.
But, I want know that where which record in data included which cluster.
For example, I make the number of 100 cluster, and i have 130,000 data, and
I obtain just 100 centroids.

Thank you.

2018-07-05 14:26 GMT+09:00 Lampros Mouselimis <notifications@github.com>:
